src/parser/monk/brewmaster/modules/constants/Mitigation.js

```diff
@@ -9,8 +9,25 @@ export const ULDIR_K = [
   9311.4, // Mythic
 ];
 
-export const MPLUS_K = 7100.1;
+export const MPLUS_K = 8467.2;
+
+export const BOD_K = [
+     null,
+   8467.2, // LFR
+     null,
+   9311.4, // Normal
+  10275.3, // Heroic
+  11390.4, // Mythic
+];
 
 export function diminish(stat, K) {
   return stat / (stat + K);
 }
+
+export function lookupK(fight) {
```

Comment:
We're probably going to want multi-raid support soon, especially with the raid release around 8.1.5

Reply:
agreed, but the only way to do that based on available info is to map `boss id -> instance` and i didn't want to tackle that yet.

Reply:
We should have the data in the raids folder. Maybe make it a part of the raid config file?

Reply:
i'll check into it :+1: